README.md

```diff
@@ -14,7 +14,16 @@ This plugin only works with embulk >= 0.8.8.
 
 ## Configuration
 
-- **uri**: [MongoDB connection string URI](http://docs.mongodb.org/manual/reference/connection-string/) (e.g. 'mongodb://localhost:27017/mydb') (string, required)
+- Connection parameters
+  One of them is required.
+  
+  - use MongoDB connection string URI
+    - **uri**: [MongoDB connection string URI](http://docs.mongodb.org/manual/reference/connection-string/) (e.g. 'mongodb://localhost:27017/mydb') (string, required)
+  - use separated URI parameters
+    - **hosts**: list of hosts. `hosts` are pairs of host(string, required) and port(integer, optional, default: 27017)
+    - **username**: (string, optional)
```

Comment:
I think `user` is better, as same as other embulk plugin like [embulk-input-jdbc](https://github.com/embulk/embulk-input-jdbc)